scripts/pancancer_classifier.py

```diff
@@ -14,20 +14,20 @@
     --folds             number of cross validation folds (defaults to 5)
     --drop              drop the input genes from the X matrix
     --copy_number       optional flag to supplement copy number to define Y
-    --filter_count      int of low count of mutation to include tissue
-    --filter_prop       float of low proportion of mutated samples per tissue
+    --filter_count      int of low count of mutation to include disease
```

Comment:
that's a good suggestion - I should specify all default parameters here for the user. Updated in next commit!